### PR TITLE
Create helloworld.py

### PR DESCRIPTION
**Pull request by JosephPobega**

### CODE REVIEW

File: lesson1/helloworld.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Hello Python world!")
```
Comment (Pobega) [inline lesson1/helloworld.py:2]: This should have a shebang line at the top, like `#!/usr/bin/env python3`, otherwise it won't know how to run!

Comment (Pobega) [reply lesson1/helloworld.py:2]: Also please include a comment at the top that describes what the file does. You use the quotes to make a multiline comment, like this:


"""
Author: Joe Pobega
Description: just a simple hello world program
"""

Comment (JosephPobega) [reply lesson1/helloworld.py:2]: ok! I will get to doing that right now. Check lesson2 for me.

**Comment by JosephPobega** [comment]
sent over the pull request. You should now be able to open the file! :)

**Pull request merged**